chapters/equations.tex

```diff
@@ -912,7 +912,7 @@ \subsection{Equations Needed for Initialization}\label{the-number-of-equations-n
 \end{lstlisting}
 
 Here we have three variables with unknown values: two dynamic variables that also are state variables, \lstinline!x1! and \lstinline!x2!, i.e., $n=2$, one output variable \lstinline!y!, i.e., $m=1$, and one input variable \lstinline!u! with known value.
-A consistent solution of the initial value problem providing initial values for \lstinline!x1!, \lstinline!x2!, \lstinline!der(x1)!, \lstinline!der(x2)!, and \lstinline!y! needs to be found.
+A consistent solution of the initialization problem requires initial values for \lstinline!x1!, \lstinline!x2!, \lstinline!der(x1)!, \lstinline!der(x2)!, and \lstinline!y! needs to be found.
 Two additional initial equations thus need to be provided to solve the initialization problem.
```

Comment:
```suggestion
A consistent solution of the initialization problem requires finding initial values for \lstinline!x1!, \lstinline!x2!, \lstinline!der(x1)!, \lstinline!der(x2)!, and \lstinline!y!.
Two additional initial equations thus need to be provided to uniquely specify the initialization problem.
```
'To be found' seemed very passive, and the second sentence was plain wrong; there's no problem solving problems with missing equations - we need the extra equations to uniquely specify the solution.

Reply:
Maybe we should be careful with saying _uniquely_?  It would be more correct to talk about _well-determined_ or _(globally) balanced_:
```suggestion
A consistent solution of the initialization problem requires finding initial values for \lstinline!x1!, \lstinline!x2!, \lstinline!der(x1)!, \lstinline!der(x2)!, and \lstinline!y!.
Two additional initial equations thus need to be provided to obtain a well-determined (i.e., globally balanced) initialization problem.
```